pyqir-generator/src/python.rs

```diff
@@ -201,6 +203,32 @@ impl BasicQisBuilder {
         BasicQisBuilder { builder }
     }
 
+    #[args(args = "*")]
+    fn call(&self, py: Python, name: String, args: &PyTuple) {
```

Comment:
Please add this to qis.pyi, or builder.pyi once you have moved the definition there. Otherwise, mypy typechecking will fail. (We should probably add mypy to CI.)